app/models/podcast_import.rb

```diff
@@ -262,9 +280,15 @@ def create_story(entry, series)
       explicit: explicit(entry[:itunes_explicit])
     )
 
-    # add the audio
-    if enclosure = enclosure_url(entry)
-      audio = version.audio_files.create!(label: 'Segment A', upload: enclosure)
+    if config[:audio] && config[:audio][entry.entry_id]
```

Comment:
So just to confirm ... if an entry_id is missing from the config, you're going to silently get a single-segment audio version.  (Breaking the validation rules).

Or does this actually throw a "not enough segments" validation error against the audio version template?